packages/inferno-animation/src/animationCoordinator.ts

```diff
@@ -0,0 +1,75 @@
+import { forceReflow } from './utils';
+
+export enum AnimationPhase {
```

Comment:
make this constant enum and typescript compiler will inline the numbers into the switch case statement :)

Reply:
WILL FIX